asserts/asserts.go

```diff
@@ -246,12 +235,12 @@ func buildAssertion(headers map[string]string, body, content, signature []byte)
 	}
 
 	if _, err := checkMandatory(headers, "authority-id"); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("assertion %v", err)
```

Comment:
It's easy to get the wording messed up here, because the person editing the error downstream won't see the call site. I suggest either turning it into a real prefix, or perhaps more easily, get a `what string` parameter into checkMandatory itself, so it can take it into account.